File: EclipseWorkspace/VorkursWilkommen/src/a62.java
public class a62{
		public static int kgv(int a,int b){
			return a*b/a51.euclid(a,b);
		}
		public static int kgv(int[] a, int index){
			if (a.length -1 >= index +1){
				return kgv(a[index],a[index++]);
			}else{
				return kgv(a[index],kgv(a,index++));
			}
		}
		public static void main(String args[]){
		
		int arr[] = SimpleIO.getRandomIntArray();
		System.out.println(kgv(arr,0));
	}
}

File: EclipseWorkspace/VorkursWilkommen/src/a62.erl
-module (a62).
-export ([kgv/1,ggt/2,kgv/2]).
ggt(A,0)->
	A;
ggt(A,B) ->
	ggt(B,round(A) rem round(B)).
kgv (A,B) ->
	A*B/ggt(A,B).
kgv([T])->
	T;
kgv([H|T]) ->
	kgv(H,kgv(T)).